src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__box container__box--1"></div>
+      <div class="container__box container__box--2"></div>
+      <div class="container__box container__box--3"></div>
+      <div class="container__box container__box--4"></div>
+      <div class="container__box container__box--5"></div>
+      <div class="container__box container__box--6"></div>
```

Comment:
Sorry, missed it last time.
No need to create additional classes. Just use `container__box` with `nth-child` pseudo-class